src/ngraph/runtime/gpu/cuda_emitter.cpp

```diff
@@ -333,6 +333,93 @@ size_t runtime::gpu::CUDAEmitter::build_pad_dynamic(const runtime::gpu::GPURunti
     return primitive_index;
 }
 
+size_t runtime::gpu::CUDAEmitter::build_reshape(const runtime::gpu::GPURuntimeContext* ctx,
+                                                const std::array<std::string, 2>& dtypes,
+                                                GPUShape input_shape,
+                                                GPUShape input_order)
+{
+    auto rank = input_shape.size();
+    std::stringstream kernel_name;
+    kernel_name << "slice_" << join(dtypes, "_") << "_r_" << rank;
```

Comment:
Missed this, @fengleitian can you update the kernel_name here? Thanks

Reply:
Let me update in another PR.

Reply:
update here:
https://github.com/NervanaSystems/ngraph/pull/1204